src/activity_handlers.cpp

```diff
@@ -1028,26 +1028,26 @@ void activity_handlers::reload_finish( player_activity *act, player *p )
         return;
     }
 
+    std::string msg = _( "You reload the %s." );
+
     if( reloadable->is_gun() ) {
         p->recoil -= act->moves_total;
 
         if( reloadable->has_flag( "RELOAD_ONE" ) ) {
             for( int i = 0; i != qty; ++i ) {
```

Comment:
I'm not certain how the message long concatenates messages with a `N x` prefix but at a minimum we'd need to duplicate that same logic here. If it prefixes on the first duplicate there's an argument for dropping the loop.